arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -7,12 +7,16 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		092A1C2722456D9500C5F230 /* DisplayWFS.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 092A1C2322456D9500C5F230 /* DisplayWFS.storyboard */; };
+		092A1C2A22456D9500C5F230 /* DisplayWFSViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 092A1C2622456D9500C5F230 /* DisplayWFSViewController.swift */; };
+		092A1C2B22456DBF00C5F230 /* DisplayWFSViewController.swift in Resources */ = {isa = PBXBuildFile; fileRef = 092A1C2622456D9500C5F230 /* DisplayWFSViewController.swift */; };
```

Comment:
`DisplayWFSViewController.swift` is being copied as part of both the Copy Bundle Resources and Copy Files phases. Since our practice is to copy source files as part of the Copy Files phase, please remove it from the Copy Bundle Resources phase.

![Screen Shot 2019-03-22 at 12 38 38 PM](https://user-images.githubusercontent.com/2257493/54848942-e3fc8900-4c9f-11e9-85e2-ca811e230b15.png)